README.md

```diff
@@ -39,7 +39,7 @@ before starting work on new features.
     Python3-compatible [virtualenv][virtualenv] for AMY and install
     dependencies from `requirements.txt`.
 
-3.  Install [yarn](https://yarnpkg.com/lang/en/docs/install), the tool that manages AMY's JavaScript and CSS dependencies. [You can install it here](https://yarnpkg.com/lang/en/docs/install).
+3.  Install [yarn](yarn), the tool that manages AMY's JavaScript and CSS dependencies. [You can install it here](yarn).
```

Comment:
If you want to use this type of links in Markdown, you need to use `[display][link]` form (note the brackets); otherwise it will point to `./link`, as it is now.